intera_interface/src/intera_interface/camera.py

```diff
@@ -98,7 +98,7 @@ def is_camera_streaming(self, camera_name):
         return False
 
     def set_callback(self, camera_name, callback, callback_args=None,
-        queue_size=10, rectify_image=True):
+        queue_size=1, buff_size=2**24, rectify_image=True):
```

Comment:
For most camera applications, users want the most recent image and not a stale older image. Also, the buffer_size would need to be scaled by the queue_size which could create a fairly large buffer.